pages/eth-unit-conversion.page.tsx

```diff
@@ -1,55 +1,70 @@
-import { ChangeEvent, Fragment, useState } from 'react';
+import { ChangeEvent, useState } from 'react';
 
 import CalculatorSvg from '../public/static/svg/calculator';
+import { Input } from '../src/components/lib/Input';
 import { ToolLayout } from '../src/layout/ToolLayout';
 import { UnitType } from '../src/lib/convertProperties';
 import { convertEthUnits } from '../src/lib/convertUnits';
 import { decodeHex } from '../src/lib/decodeHex';
 import { unitSchema } from '../src/misc/unitSchema';
 
-type State = { wei: string; gwei: string; eth: string };
+type EthUnitConversionState = Record<
+  UnitType,
+  { value: string; error?: string }
+>;
+
+const powers = {
+  wei: -18,
+  gwei: -9,
+  eth: -1,
```

Comment:
Not sure if we want to change that but `10^-1` is an unintuitive way to write `1`. Although coherent.
<img width="472" alt="Screen Shot 2022-04-03 at 3 33 21 PM" src="https://user-images.githubusercontent.com/6204356/161430606-15fdd8ef-fe8b-4bb5-9e5d-3e829802ada4.png">

Reply:
Yeah, I’ve been thinking about it. Was this in the designs? Shouldn’t we maybe just skip it?